### PR TITLE
add breakoutnotes

### PR DESCRIPTION
**Pull request by balintstewart77**
adding notes from the March quarterly meeting

### CODE REVIEW

**Comment by manics** [comment]
I've updated this PR, should be ready for review now!

File: docs/events/wg_workshops/2024-03-14-march-meeting/index.md [hunk 1]
```diff
@@ -124,3 +132,43 @@ We had a successful meeting on 27th Feb where several topics around implementati
...
+**Next steps**
+
+- Slides and break out notes to be circulated following the meeting.
+- Interested in working with us to provide requirements and/or test a researcher passport solution? Please contact rachel.tesfaye@hdruk.ac.uk
+
+## Breakout Session 2, Room 3: TRE at King’s: Deployment & Egress Simulation
+
+## Breakout Session 2, Room 4: Trenity demo - an in house tool for managing users, projects and file transfers in TREs
+
+**Summary**
```
Comment (Davsarper) [inline docs/events/wg_workshops/2024-03-14-march-meeting/index.md:162]:
```suggestion
```
Empty, better to take out?

File: docs/events/wg_workshops/2024-03-14-march-meeting/index.md [hunk 2]
```diff
@@ -124,3 +132,43 @@ We had a successful meeting on 27th Feb where several topics around implementati
...
+
+**Next steps**
```
Comment (Davsarper) [inline docs/events/wg_workshops/2024-03-14-march-meeting/index.md:164]:
```suggestion
```
Empty, better to take out?

**Review by Davsarper — DISMISSED** [review]
Minor suggestion that should not stop merging, feel free to apply and merge or dismiss and merge

**Pull request merged**